kong/runloop/cache_prewarm.lua

```diff
@@ -23,6 +29,12 @@ local function cache_prewarm_single_entity(entity_name)
     end
 
     local cache_key = dao:cache_key(entity)
+
+    local ok, err = kong.cache:safe_set(cache_key, entity)
+    if not ok then
+      return nil, err
+    end
+
```

Comment:
Now that we are directly setting those values in L2, there is no point in calling `:get()` like we do below anymore. Unless I missed something?

Reply:
It will warm L1, but perhaps there is `lock` involved? Is there? That may cause problems in `init_worker`?

Reply:
I don't see any benefits in warming up L1 here. There would COW benefits be if this was being executed in `init`, but that isn't the case as-is. This will likely slow down startup due to the worker mutexes.

Reply:
@thibaultcha is there a mutex when reading value from L2? I am pretty sure there will not. The callback is never called. I changed return entity to error('should never be called').

Reply:
here: https://github.com/Kong/kong/pull/4408/commits/6085ac2065eb7817c4c5b8847d168cc105a3d236#diff-f88344d1a956d42966b81639f8302e26R39

Reply:
Ok, that's fine. Nonetheless, warming up L1 has close to zero benefits here, unless L1 serialization would be doing I/O or something, which it isn't.

Reply:
Agreed. L1 cache is really for reducing GC pressure from having to recreate objects every time L2 hits. Which in this scenario, provides close to no benefits from a performance perspective.

Reply:
It will do it in first need, which could be argued with same sentence as well, e.g. zero to no benefit? Why not do it if it is cheap as hell to do it before a first request comes in? I mean what is the downside, it should add almost nothing to warmup time? I surely can comment out those 3 lines if that is a real issue.